client/src/containers/Filters/styles.module.scss

```diff
@@ -15,6 +17,14 @@
 		.actionWrapper {
 			margin-left: 10px;
 			flex: 0 0 auto;
+			.createBtn {
```

Comment:
can you please just @include primary-btn cause if we wanna change primary buttons all over the project we can just change mixin